tutorials/1_basic_square/code/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: basic_square
+description: Example code for the the tutorial (Basic Rendering a simple square on the screen)
+
+
```

Comment:
nit: remove this whitespace?

Reply:
did you also remove that publish:none line? that made sense in this case right?

Reply:
You asked me to remove it on a previous review 🤔 

Or maybe I didn't understood? Either way I have re added it.